web/html/src/manager/content-management/shared/business/filters.enum.ts

```diff
@@ -106,6 +111,12 @@ export const clmFilterOptions: ClmFilterOptionsEnumType = {
     entityType: filterEntity.PACKAGE,
     matchers: [filterMatchers.EQUALS],
   },
+  PROVIDES_NAME: {
+    key: "provides_name",
+    text: t("Contains Package Provides Name"),
+    entityType: filterEntity.PACKAGE,
+    matchers: [filterMatchers.PROVIDES_NAME],
+  },
```

Comment:
Why not just "Provides" instead of "Provides Name"? Is any other thing possible in the future?

Reply:
Yes, every dependency has also a version component. We can have more filters with version compare in future